stellargraph/datasets/datasets.py

```diff
@@ -849,3 +850,55 @@ def load(self):
         )
 
         return StellarGraph(nodes=nodes, edges=edges, edge_weight_column="time"), edges
+
+
+@experimental(reason="the data isn't downloaded automatically", issues=[9999])
```

Comment:
The 9999 in my suggestion here and in `test-demo-notebooks.sh` was a placeholder that needs to be replaced by a real issue number. At the moment, this is creating a link like https://github.com/stellargraph/stellargraph/issues/9999, which doesn't currently exist. Could you open an issue about this and then replace the 9999 here and in `test-demo-notebooks.sh`?

Reply:
Ah. Ok! I'll fix this.

Reply:
Great! Reminder:

```suggestion
@experimental(reason="the data isn't downloaded automatically", issues=[1303])
```